### PR TITLE
Cd review/feature1

### PR DESCRIPTION
**Pull request by simonysm**
pls. help review.

### CODE REVIEW

**Comment by simonysmwork** [comment]
very good. can merged into main.

File: hello4.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hello4
```
Comment (simonysmwork) [inline hello4.txt:1]: very good

File: hello2.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+hello2
```
Comment (simonysmwork) [inline hello2.txt:1]: very good

File: hello4.txt (hunk 1 above)
Comment (simonysm) [reply hello4.txt:1]: thx

File: hello2.txt (hunk 2 above)
Comment (simonysm) [reply hello2.txt:1]: hello2 is good

**Review by simonysmwork — APPROVED** [review]

**Pull request merged**